### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by gadhagod**
This PR fixes [lint errors](https://github.com/rockset/rockset-js/actions/runs/929362827) and updates the README of rscode to have features added in #18 and #19.

On a side note, I think [there is a way to have `build|lint|test` on pull requests](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request). That would be super useful, to prevent similar errors in the future.

### CODE REVIEW

File: packages/rscode/README.md [hunk 1]
```diff
@@ -44,27 +45,45 @@ Then, open `myLambda.sql` in VSCode. You should confirm that the language of you
...
 
-### Formatting documents
+### Executing and Validating Queries
+You can execute and validate simple queries directly in VSCode. Open the Command Palette in VSCode (<kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd> or <kbd>CMD</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd>) and execute the command `Execute Rockset Query` or `Validate Rockset Query`. At this point, it is not possible to run parameterized queries directly from VSCode.
 
+### Adding Documents
+You can add documents with the [Write API](https://docs.rockset.com/write-api/) directly from VSCode. With a JSON or YAML file open, execute the command `Add Rockset Document(s)`. This command takes either a list of documents or a single document.
+
+### SQL Documentation
+You can hover over any SQL function and get it's parameters and description, along with a link to Rockset's documentation.
```
Comment (scottwmorris) [inline packages/rscode/README.md:55]: "it's" -> "its"

**Review by scottwmorris — COMMENTED** [review]
minor grammar nit, then I'll merge, thanks @gadhagod !

**Comment by gadhagod** [comment]
> minor grammar nit, then I'll merge, thanks @gadhagod !

Fixed that up @scottwmorris. Glad I could help.

**Comment by gadhagod** [comment]
Pinging @scottwmorris.

**Pull request merged**